impeller/entity/geometry/stroke_path_geometry.cc

```diff
@@ -128,9 +128,9 @@ StrokePathGeometry::JoinProc StrokePathGeometry::GetJoinProc(Join stroke_join) {
                                PathBuilder::kArcApproximationMagic * alignment *
                                dir;
 
-        auto arc_points = CubicPathComponent(start_offset, start_handle,
-                                             middle_handle, middle)
-                              .CreatePolyline(scale);
+        std::vector<Point> arc_points;
+        CubicPathComponent(start_offset, start_handle, middle_handle, middle)
```

Comment:
Can the reserve size be calculated here?

Reply:
These are small and shouldn't result in lots of reallocation. I'd feel more comfortable having a benchmark showing this before trying to place a number in it.

Reply:
I ask because I'm turning on a linter for this in https://github.com/flutter/engine/pull/47868.  I don't think it will flag this though since it can't trivially know what the reserve size is.

Reply:
Reserving incorrect sizes can make things worse. If we don't know let's use the standard vec reallocation startegy